doc/installation_instructions/platform/perlmutter.rst

```diff
@@ -10,24 +10,32 @@ To install SmartSim on Perlmutter, follow these steps:
 
 .. code:: bash
 
-    module load conda
+    module load conda cudatoolkit/12.2 cudnn/8.9.3_cuda12 PrgEnv-gnu
     conda create -n smartsim python=3.11
     conda activate smartsim
 
-**Step 2:** Install SmartSim in the conda environment:
+**Step 2:** Build the SmartRedis C++ and Fortran libraries:
+
+.. code:: bash
+
+    git clone https://github.com/CrayLabs/SmartRedis.git
+    cd SmartRedis
+    make lib-with-fortran
+    pip install .
+
+**Step 3:** Install SmartSim in the conda environment:
```

Comment:
Before we pip install SmartSim I think we want to `cd` out of the SmartRedis dir